Dockerfile

```
@@ -2,22 +2,29 @@
 
 ARG FLAVOR=${TARGETARCH}
 
-ARG ROCMVERSION=6.1.2
+ARG GOVERSION=1.23.4
+ARG ROCMVERSION=6.3.2
```

```suggestion
ARG ROCMVERSION=6.3.3
```

Again v6.3.3 is already out in the meanwhile: https://rocm.docs.amd.com/en/docs-6.3.3/

---

Once I've verified the version bumps don't cause any regressions, I'll rebase on #8211 with 6.3.3